<?xml version="1.0"?>

<launch>

  <node name="power_node"  pkg="takasako_sps" type="scpi_tcp_client_node" output="screen" />

  <node name="forces_node" pkg="cfs_sensor" type="cfs_node"  output="screen"/> 

  <include file="$(find rosserial_stm)/launch/rosserial.launch" />

  <node name="motor_test_node" pkg="motor_test" type="motor_test_node"  output="screen">
    <param name="pwm_incremental_value" value="25" />
    <param name="duration" value="3.0" />
    <param name="min_pwm_value" value="1100" />
    <param name="max_pwm_value" value="1900" />

    <remap from="/forces" to="/cfs/forces" />
    <remap from="/power_pwm" to="/pwm_test" />

  </node>
</launch>
